### DISCUSSION TITLE
How to upgrade and keep RSS feeds?

### DISCUSSION THREAD

**Discussion by drt1** [Novik/ruTorrent#2471] [Q&A] [answered]
Dear folks,
when upgrading rutorrent, I usually replace the old rutorrent folder with the new one and make my changes to config.php
But I lose all of my RSS feeds and RegEx queries for the feeds.

Where are the RSS feeds stored and how can move them from old install to new one?

thanks

**Comment by buggsi** [comment, marked as answer]
They're stored in `rutorrent/share/users/<your username>/settings/rss/cache` (location may vary, but it's the `settings/rss/cache` folder), the files `data` `filters` `history` and `info`. You can backup and restore those files and your rss feeds will be right as you left them.

**Comment by drt1** [comment]
Thank you very much, that did the trick.
